sw/simulator/nps/nps_fdm_jsbsim.cpp

```diff
@@ -597,7 +604,8 @@ static void init_jsbsim(double dt)
   }
 
   // initial commands to zero
-  feed_jsbsim(0.0, 0.0, 0.0, 0.0);
+  double init_commands[5] = {0.0};
```

Comment:
This results in a segfault when you only have 4 commands (like for quadrotor models)...
The commands_nb must match the number of actuator names in feed_jsbsim